ci-operator/step-registry/ipi/conf/vsphere/dns/ipi-conf-vsphere-dns-commands.sh

```diff
@@ -0,0 +1,92 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+set -x
```

Comment:
we generally don't do this to avoid leaking pull secrets and other internal details.  Can we drop this before the PR lands?